File: backend/src/performance/002-replace_substring/rs.js
"use strict"

function rs() {
    for (var i=0; i<process.argv[2]; i++) {
        var string = "Salida de emergencia",
            pattern = "Salida",
            substitution = "Entrada";
        var result = string.replace(pattern, substitution);
    }

}

if (require.main == module){
    rs()
}

File: backend/src/performance/002-replace_substring/rs.rb


def rs()
    (0..Integer(ARGV[0])).step(1) do |_|
        string = "Salida de emergencia"
        pattern = "Salida"
        substitution = "Entrada"
        result = string.sub(pattern, substitution)
    end
end

if __FILE__ == $0
    rs()
end
